real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/planners/WayPointUtil.java

```diff
@@ -42,17 +42,20 @@
 
 		File waypointFile = new File(filename);
 		Gson translator = new GsonBuilder().serializeSpecialFloatingPointValues().create();
-		Scanner fileReader = new Scanner(new FileInputStream(waypointFile), "UTF-8");
-
-		while (fileReader.hasNextLine()) {
-			String nextline = fileReader.nextLine();
-			if (nextline.equals("")) {
-				break;
+		try{
+			Scanner fileReader = new Scanner(new FileInputStream(waypointFile), "UTF-8");
+			while (fileReader.hasNextLine()) {
+				String nextline = fileReader.nextLine();
+				if (nextline.equals("")) {
+					break;
+				}
+				waypoints.add(translator.fromJson(nextline, GpsMeasurement.class));
 			}
-			waypoints.add(translator.fromJson(nextline, GpsMeasurement.class));
-		}
-		
 		fileReader.close();
+		
+		}catch(java.io.FileNotFoundException e){
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'java.io.FileNotFoundException' due to existing import 'java.io.FileNotFoundException'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2662564663/issues/source?bid=2938665&fileBranchId=3246677#l56)